_episodes/01-intro.md

```diff
@@ -128,7 +127,7 @@ So let's look at the REPL (read-evaluate-print loop) in more detail. Notice that
        interpret as the program sees fit 
 3. Print the output produced by the program
 
-and then print the prompt and wait for you to enter another command.
+and then print the prompt and wait for a new command.
```

Comment:
Modified version reduces clarify. Where does the new command come from?

Reply:
okay